src/Servers/Kestrel/Core/src/Internal/Http/Http1MessageBody.cs

```diff
@@ -52,7 +52,7 @@ protected override Task OnConsumeAsync()
                     }
                 }
             }
-            catch (BadHttpRequestException ex)
+            catch (Microsoft.AspNetCore.Http.BadHttpRequestException ex)
```

Comment:
Using statement?

Reply:
I can use a using directive here but it must go inside the namespace again.